### PR TITLE
lab 8

### PR DESCRIPTION
**Pull request by vnikiforova**

### CODE REVIEW

File: lecture_notes/08 collections/1-array.js [hunk 1]
```diff
@@ -1,8 +1,50 @@
...
 const ages = [10, 12, 15, 15, 17, 18, 18, 19, 20];
+const total = ages.push(21, 21, 22);
 
-const first = ages[0];
-const last = ages[ages.length - 1];
+console.log(ages); // [10, 12, 15, 15, 17, 18, 18, 19, 20, 21, 21, 22]
+
+
+// POP
+'use strict';
```
Comment (tshemsedinov) [inline lecture_notes/08 collections/1-array.js:11]: `'use strict';` should be always at the beginning of the file or beginning of the function

File: lecture_notes/08 collections/4-concat.js [hunk 2]
```diff
@@ -13,10 +9,19 @@ const concat = (arr1, arr2) => {
...
 const schoolAges = [10, 12, 15, 15];
 const studentAges = [17, 18, 18, 19, 20];
 const ages = concat(schoolAges, studentAges);
 const { first, last } = getFirstAndLast(ages);
 console.log({ first, last });
+
+
+// Exaple CONTACT
+const ArrLowerCase1 = ['h', 'e', 'l', 'l', 'o'];
+const ArrLowerCase2 = ['w', 'o', 'r', 'l', 'd'];
```
Comment (tshemsedinov) [inline lecture_notes/08 collections/4-concat.js:21]: Rename to `greeting` and `target`

File: lecture_notes/08 collections/4-concat.js [hunk 3]
```diff
@@ -13,10 +9,19 @@ const concat = (arr1, arr2) => {
...
+
+const UpperCase1 = ArrLowerCase1.map(name => name.toUpperCase());
+const UpperCase2 = ArrLowerCase2.map(name => name.toUpperCase());
```
Comment (tshemsedinov) [inline lecture_notes/08 collections/4-concat.js:24]: Code duplication

**Review by tshemsedinov — CHANGES_REQUESTED** [review]
- Don't write python code in js files, make copies
- Improve code style, fix all linter errors (see tab `Checks`)

**Comment by vnikiforova** [comment]
okey

**Pull request closed without merging**